tests/misc/print_exception.py

```diff
@@ -31,7 +31,7 @@ def print_exc(e):
 
 # basic exception message
 try:
-    1/0
+    '1'+1
```

Comment:
How about `1//0`, to try and make a minimal change?

Reply:
Good idea!